tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -560,131 +332,29 @@ stages:
         - name: testPackage
           value: '@fluid-private/test-end-to-end-tests'
           readonly: true
-        - name: escapedTestPackage
-          value: ${{ replace(replace(variables.testPackage, '@', '' ), '/', '-') }}
-          readonly: true
```

Comment:
This logic got absorbed by the `include-test-perf-benchmarks-install-package.yml` template